src/api/app/models/project_log_entry.rb

```diff
@@ -67,6 +67,8 @@ def additional_info
   def self.username_from(payload)
     USERNAME_KEYS.each do |key|
       username = payload[key]
+      # FIXME: Why is commenter `id`` when everything else is `login`?
+      username = User.find_by(id: payload[key]) if key == 'commenter'
```

Comment:
This returns the user object, not the username, right?

Reply:
Hm. Can't remember why I did it this way :-/